website/package.json

```diff
@@ -16,8 +16,8 @@
     "lint:fix": "eslint ./src/ --fix"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.0",
-    "@docusaurus/preset-classic": "2.0.0-beta.0",
+    "@docusaurus/core": "2.0.0-beta.4",
```

Comment:
Please revert to the version currently being used.

Reply:
Done!